src/site/_includes/content/lighthouse-performance/improve.njk

```diff
@@ -0,0 +1,20 @@
+The best way to improve your performance metrics
+is to consider your page load speed holistically.
+Any increase in load speed will improve
+your overall Lighthouse Performance score
+_and_ most individual metrics.
+
+Use the **Opportunities** section of your Lighthouse report
+to determine which improvements will have the most value for your page.
+The more significant the opportunity,
+the greater the effect it will have on your Performance score.
+For example, the Lighthouse screenshot below shows that
+[eliminating render-blocking resources](/render-blocking-resources)
+will yield the biggest time savings for the audited page's load speed:
+
+<figure class="w-figure">
+  <img class="w-screenshot w-screenshot--filled" src="/images/includes/opportunities.png" alt="Lighthouse: Opportunities section">
```

Comment:
I think I'm getting a 404 on this image.

Reply:
Oh, that's part of an include for the `lighthouse-performance` collection that I forgot to include here. Fixed.